docs/edgeql/paths.rst

```diff
@@ -241,6 +241,61 @@ that the type name (in this case ``User``) doesn't need to be specified.
 
 .. _ref_eql_paths_link_props:
 
+Using backlinks to find items
+-----------------------------
+
+Another handy use for backlinks is using them to filter and find items
+such as during an update.
+
+Take the following schema with a ``Musician`` and an ``Instrument`` type:
+
+.. code-block:: sdl
+
+    type Musician {
+      required name: str;
+      plays: Instrument;
+    }
+
+    type Instrument {
+      required description: str;
+      required owned: bool;
+    }
+
+First insert an ``Instrument``, a ukulele:
+
+.. code-block:: edgeql-repl
+
+    db> insert Instrument { 
+    ...   description := "Tenor Stratocaster ukulele", 
+    ...   owned := false 
+    ... };
+    {default::Instrument {id: 0d3b3bc8-6013-11ee-8bab-6b7c6f4da55a}}
+
+Then a ``Musician`` with a link to the instrument. (Note the ability to cast
+from a ``uuid`` to an object type, which was added in EdgeDB 3.0!)
```

Comment:
Love that you called back here. I would cross-link to the docs on that as well. (This might require adding a label if that section doesn't already have one.)